Simbody/tests/TestHuntCrossleyForce_smooth.cpp

```diff
@@ -145,6 +144,68 @@ void testForces() {
                 expectedForce[sphere.getMobilizedBodyIndex()][1]);
         }
     }
+
+    // Do it with different ground directions and offsets
+    Vector_<SpatialVec> expectedForce2(matter.getNumBodies());
+    
+    for (int i = 0; i < 20; i++) {
+        for (Real offset = -0.1; offset <= 0.1; offset += 0.01) {
+            Vec3 normal_rand((double)(rand() % 100 - 50)  / RAND_MAX, 
+                (double)(rand() % 100 - 50) / RAND_MAX, 
+                (double)(rand() % 100 - 50) / RAND_MAX);
+            normal_rand = normal_rand / normal_rand.norm();
+            hc_smooth.setGroundPlane(normal_rand, offset);
+            Vec3 loc_inG(0.23, 0.10, 0.42); // random location of the sphere in 
+            // the space, in Ground frame
+            sphere.setQToFitTranslation(state, loc_inG);
+            Vec3 vel_inG(0.34, -0.65, -0.48); // random velocity of the sphere 
+            // in the space, in Ground frame
+            sphere.setUToFitLinearVelocity(state, vel_inG);
+            system.realize(state, Stage::Dynamics);
+
+            Vec3 contactPoint = loc_inG - normal_rand*radius;
+            const Real depth = -(dot(contactPoint,normal_rand)-offset);
```

Comment:
So this line doe sthe same job as:
`const Real Indentation = - GroundPlane.getDistance(contactPointPos);`
in the source file?

Reply:
Yes, the idea would be that in the test we are calculating the same, but using different functions, right? Otherwise, it would be the same code two times

Reply:
Yes although I think the point is more to make sure that all is fine when comparing to `system.getRigidBodyForces`